source/agora/node/Validator.d

```diff
@@ -61,6 +61,13 @@ import core.time;
 
 public class Validator : FullNode, API
 {
+    protected enum ValidatorTimers
+    {
+        NameRegistration = super.FullNodeTimers.max + 1,
+        PreImageReveal,
+        PreImageCatchup
+    }
```

Comment:
`typeof(super)`

Weird, you should get a deprecation for this usage :|
Also, if you use `TimerIdx` or similar, you can just shadow it and re-use the same name.
Also, trailing comma